src/collections/_documentation/error-reporting/configuration/drain-example/native.md

```diff
@@ -6,3 +6,6 @@ sentry_shutdown();
 
 Calling `sentry_shutdown()` before exiting the application is critical to avoid
 data loss.
+
+After shutdown the client cannot be used any more so make sure to only do that
```

Comment:
```suggestion
After shutdown, the client cannot be used anymore, so make sure to do that
```

Reply:
Same concern re: "only".

"make sure to do something _only_ [at this given time]"

vs

"make sure to do something [at this given time]"

Reply:
How about:

After shutdown, the client cannot be used anymore. It's important to only call `sentry_shutdown()` immediately before shutting down the application.

Reply:
Suggestion applied, thank you!